GPU/GLES/GLES_GPU.cpp

```diff
@@ -731,6 +736,10 @@ void GLES_GPU::BeginFrameInternal() {
 		dumpThisFrame_ = false;
 	}
 
+	// Save the cache from time to time. TODO: How often?
```

Comment:
Might be even better to simply save the cache when going to the menu, or something. Saving the cache will be pretty fast no matter what as it's just a few IDs.